amplify/backend/api/colonycdapp/schema/balance.graphql

```diff
@@ -93,4 +114,47 @@ type TokenExchangeRate @model {
   Exchange currency
   """
   marketPrice: [MarketPrice]!
+}
+
+"""
+Represents a table for caching the total balance
+"""
+type CacheTotalBalance @model {
+  """
+  Unique identifier
+  """
+  id: ID!
+  """
+  Address of the colony on the blockchain
+  """
+  colonyAddress: ID! @index(name: "byColonyAddress", queryField: "cacheTotalBalanceByColonyAddress", sortKeyFields: ["date"] )
+  """
+  Domain id within a colony
+  """
+  domainId: String
+  """
+  The timeframe type (day/week/month) for which we want to cache the in/out token balance
+  Defaults to TimeframeType.DAILY
+  """
+  timeframeType: TimeframeType!
+  """
+  The timeframe period for which we want to cache the in/out token balance
+  """
+  timeframePeriod: Int!
+  """
+  Request timestamp
+  """
+  date: AWSDateTime!
+  """
+  The total in balance in USDC
+  """
+  totalUSDCIn: String
+  """
+  The total out balance in USDC
+  """
+  totalUSDCOut: String
+  """
+  The total balance (totalIn - totalOut) in USDC
+  """
+  totalUSDC: String
```

Comment:
just a random question: is it okay that we use USDC as a constant here? shouldn't it be something like "totalBase" + "baseCurrency" so we can change it in the future? Or it is fine to have permanent conversion to USDC?